napari/_tests/test_viewer.py

```diff
@@ -137,3 +137,61 @@ def test_changing_theme(viewer_factory):
 
     with pytest.raises(ValueError):
         viewer.theme = 'nonexistent_theme'
+
+
+@pytest.mark.parametrize('layer_class, data, ndim', layer_test_data)
+def test_roll_traspose_update(viewer_factory, layer_class, data, ndim):
+    """ Controlling that translation and rolling preserve correct sequence
+    of translate and scaling values.
+    """
+
+    def check_view_consistency(layer, viewer, transf_dict):
+        """ Utility function for doing the checks.
+        """
```

Comment:
```suggestion
        """Utility function for doing the checks."""
```

It would also be nice to describe *what* checks the function is doing.

Reply:
Also, does this need to be a nested function?

Reply:
added doc string and moved to test utils